js/lib/ui.js

```diff
@@ -73,6 +75,11 @@ elgg.ui.toggles = function(event) {
 			$elem.toggle();
 		}
 	});
+
+	$(document).trigger('elgg.ui.toggle', [{
+		$toggler: $this,
+		$target: $target
+	}]);
```

Comment:
I replaced this with trigger_hook. What's the big win of triggering on the toggler element?